### ISSUE TITLE
Should provide command flags for redis 6

### ISSUE THREAD

**Issue by TheCycoONE** [brandur/redis-cell#51]
I've noticed that the list of flags returned by `COMMAND` for cl.throttle in redis 6.x is empty. There is a new CreateCommand function for Redis 6 which takes a list of command flags that apply to the command. I think @write and @fast apply.

https://redis.io/topics/modules-api-ref#coderedismodulecreatecommandcode

This may be related to issues I'm having with redis-cell and lettuce on redis 6, but I'm not sure: https://github.com/lettuce-io/lettuce-core/issues/1327

**Comment by TheCycoONE** [comment]
Looking at the code, the flag list is provided with write, which makes me wonder if there is a bug instead?
```
 72) 1) "cl.throttle"
     2) (integer) -1
     3) 1) write
     4) (integer) 0
     5) (integer) 0
     6) (integer) 0
     7) (empty array)
```

**Comment by brandur** [comment]
Interesting — I hadn't taken much of a look at Redis 6 yet, and this is the first time I've seen these flags.

There's no documentation about them in [`COMMAND`](https://redis.io/commands/command), but they do appear in the example run near the bottom of the page. They do get a [brief mention in code](https://github.com/redis-io/redis/blob/unstable/src/server.c#L161-L166) that suggests they're used for ACL categorization.

I'm not sure there's much that can be done for redis-cell though. The exported `CreateCommand` API from Redis for Modules still only takes the original arguments, and I couldn't find a new version of it. I suspect that this is a relatively new feature that is not supported for Redis Modules yet, and not even well-described in the core yet.

**Comment by TheCycoONE** [comment]
Yes, I noticed due to the linked bug in lettuce handling empty arrays, but wasn't able to find more information. I originally confused them with the flags under 3 when I opened the ticket; and backtracked when I found you were already providing those and it didn't help.

There's some mention of a new module api in an early announcement about 6 but I couldn't find anything about that either. I'll ask around.
